pyuvdata/miriad.py

```diff
@@ -29,7 +30,8 @@ def _pol_to_ind(self, pol):
         return pol_ind
 
     def read_miriad(self, filepath, correct_lat_lon=True, run_check=True,
-                    check_extra=True, run_check_acceptability=True, phase_type=None):
+                    check_extra=True, run_check_acceptability=True, phase_type=None,
+                    antpairs=None, pols=None, times=None):
```

Comment:
I'd like to see this interface match the select interface, so include these options:
antenna_nums, antenna_names, ant_str, ant_pairs_nums, frequencies, freq_chans, times, polarizations, blt_inds

If some of these can't be supported, we should document why, but we should definitely use the same keywords with the same functionality when possible. (If you think the current select interface should be modified we can discuss that, but I'd really like there to be consistency.)

Reply:
yup for sure, i'll try to make that happen